### PR TITLE
[stable/verdaccio] Support Kubernetes v1.16 API

### PR DESCRIPTION
**Pull request by damlys**
Hi @etiennetremel :wave: Hope we'll deal with that quickly :smile::+1: 

#### What this PR does / why we need it:

Updates resource's versions to support the Kubernetes v1.16 API

Currently the chart cannot be installed on a Kubernetes v1.16 cluster. For more details see:
https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/

#### Checklist

- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Review by etiennetremel — APPROVED** [review]

**Comment by damlys** [comment]
Hello @lachie83, @scottrigby

It's my first PR to this repo and I'm wondering if everything is ok? :smile: Should I do something or just wait for your approve?

**Review by Locus99 — COMMENTED** [review]
This change did not work for my group until I added a selector to the deployment spec:
```
  selector:
    matchLabels:
      app: {{ template "verdaccio.name" . }}
      release: {{ .Release.Name }}
```

**Comment by damlys** [comment]
@Locus99 thanks, I have added that :+1:

**Comment by damlys** [comment]
Hi @scottrigby

We are still waiting for `lgtm` label. Can you check this PR? Everything looks ok now

**Comment by damlys** [comment]
I'm closing this PR. Issue is resolved by https://github.com/helm/charts/pull/18329

**Pull request closed without merging**
